ee/codegen/src/__test__/nodes/inline-prompt-node-retry.test.ts

```diff
@@ -0,0 +1,244 @@
+import { Writer } from "@fern-api/python-ast/core/Writer";
+import { v4 as uuidv4 } from "uuid";
+import { beforeEach, describe, expect, it } from "vitest";
+
+import { workflowContextFactory } from "src/__test__/helpers";
+import { inputVariableContextFactory } from "src/__test__/helpers/input-variable-context-factory";
+import { inlinePromptNodeAdornmentsFactory } from "src/__test__/helpers/node-data-factories";
+import { createNodeContext, WorkflowContext } from "src/context";
+import { InlinePromptNodeContext } from "src/context/node-context/inline-prompt-node";
+import { InlinePromptNode } from "src/generators/nodes/inline-prompt-node";
+
+describe("InlinePromptRetryNode", () => {
+  let workflowContext: WorkflowContext;
+  let writer: Writer;
+
+  beforeEach(() => {
+    workflowContext = workflowContextFactory();
+    writer = new Writer();
+
+    workflowContext.addInputVariableContext(
+      inputVariableContextFactory({
+        inputVariableData: {
+          id: "90c6afd3-06cc-430d-aed1-35937c062531",
+          key: "text",
+          type: "STRING",
+        },
+        workflowContext,
+      })
+    );
+  });
+
+  const promptInputBlockTypes = [
+    "JINJA",
+    "CHAT_MESSAGE",
+    "FUNCTION_DEFINITION",
+    "VARIABLE",
+    "RICH_TEXT",
+  ] as const;
+
+  describe.each(promptInputBlockTypes)("%s block type", (blockType) => {
```

Comment:
[nit] we could simplify our testing and this PR tremendously by cutting out these tests.

Retry node is agnostic to prompt blocks used, something we already have coverage on. So we don't gain anything by testing retry nodes on every combination of prompt blocks

Reply:
Actually, going to mark this `[blocking]` as it will simplify reviewing the tests we _do_ want to include